hehe

mithunvoe opened the pull request.

mithunvoe committed:
hehe

ninadgns commented:
Resolve these issues before merging

ninadgns commented on app/dashboard/notes/page.tsx:
```diff
@@ -0,0 +1,727 @@
+                                  tbody: ({node, ...props}) => <tbody {...props} />,
+                                  tr: ({node, ...props}) => <tr className="border-b border-border" {...props} />,
+                                  th: ({node, ...props}) => <th className="px-4 py-2 text-left font-semibold text-foreground" {...props} />,
+                                  td: ({node, ...props}) => <td className="px-4 py-2 text-foreground" {...props} />,
```
use markdown parser

ninadgns committed:
fix: resolve Copilot review comments and linting errors

- Add aria-label to delete button for accessibility
- Fix markdown component props to use children instead of node
- Replace all 'any' types with proper TypeScript types (unknown with type guards)
- Fix React Hook dependency warning by using useCallback for refreshSelectedFile
- Fix variable declaration order to resolve TypeScript errors

ninadgns committed:
fix: replace 'any' type with 'unknown' in NoteContent metadata
